bc_obps/registration/tests/utils/bakers.py

```diff
@@ -44,14 +44,19 @@ def contact_baker():
     return baker.make(Contact)
 
 
-def operator_baker():
-    return baker.make(
-        Operator,
-        bc_corporate_registry_number=generate_random_bc_corporate_registry_number(),
-        business_structure=BusinessStructure.objects.first(),
-        physical_address=address_baker(),
-        website='https://www.example-operator.com',
-    )
+def operator_baker(custom_properties=None):
```

Comment:
My tests were getting repetitive to the point of sonarcloud fails, so tweaked these functions to allow for DRYer test code